src/scripts/extract_variables.py

```diff
@@ -0,0 +1,298 @@
+import argparse
```

Comment:
Missing description of what this does:

```suggestion
"""Extraction of survey questions.

Usage:
    uv run extract_variables.py [--input INPUT_FILE] [--output OUTPUT_FILE]
"""

import argparse
```